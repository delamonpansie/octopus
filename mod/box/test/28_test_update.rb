#!/usr/bin/ruby
# encoding: ASCII

$: << File.dirname($0) + '/lib'
require 'run_env'

RunEnv.connect_eval do
  insert [1, "\0\1", "\0\0\0\1", "\0\0\0\0\0\0\0\1"]
  update_fields 1, [1, :add, "aa"] , [2, :add, "aaaa"], [3, :add, "aaaaaaaa"]
  select 1
end
