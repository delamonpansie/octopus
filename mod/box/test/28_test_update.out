# box.insert([1, "\x00\x01", "\x00\x00\x00\x01", "\x00\x00\x00\x00\x00\x00\x00\x01"])
1

# box.update_fields(1, [1, :add, "aa"], [2, :add, "aaaa"], [3, :add, "aaaaaaaa"])
1

# box.select(1)
[["\x01\x00\x00\x00", "ab", "aaab", "aaaaaaab"]]

